modules/distr-tracing-architecture.adoc

```diff
@@ -25,6 +25,21 @@ This module included in the following assemblies:
 
 ** *Jaeger Console* – With the {JaegerName} user interface, you can visualize your distributed tracing data. On the Search page, you can find traces and explore details of the spans that make up an individual trace.
 
+* *{TempoName}* - This component is based on the open source link:https://github.com/grafana/tempo/[Grafana Tempo project].
```

Comment:
```suggestion
* *{TempoName}* - This component is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo project].
```

Link to grafana.com instead of github.com

Reply:
Thank you, applying.